src/pages/test-site3.js

```diff
@@ -0,0 +1,57 @@
+import {useState} from 'react';
+
+export default function Screen() {
+	const [list, setList] = useState([]);
+
+	function _onCreate(item) {
```

Comment:
Why is this function name beginning with an '_'  ?

Reply:
this shouldnt have been in the cr at all thank you